plugins/copr.py

```diff
@@ -502,22 +506,29 @@ def _runtime_deps_warning(self, copr_username, copr_projectname):
         if not runtime_deps:
             return
 
-        print(_("Maintainer of the enabled Copr repository decided to make\n"
-                "it dependent on other repositories. Such repositories are\n"
-                "usually necessary for successful installation of RPMs from\n"
-                "the main Copr repository (runtime dependencies provided).\n\n"
-
-                "Be aware that the note about quality and bug-reporting\n"
-                "above applies here too, Fedora Project doesn't control the\n"
-                "content. Please review the list:"))
-        dep_idx = 1
-        for repo_id in runtime_deps:
-            print("\n{0}. [{1}]".format(dep_idx, repo_id))
-            print("   baseurl={0}".format(repo.cfg.getValue(repo_id, "baseurl")))
-            dep_idx += 1
-
-        msg = _("\nThese repositories have been enabled automatically."
-                "\nDo you want to keep them enabled?")
+        msg = _(
+            "\nMaintainer of the enabled Copr repository decided to make\n"
+            "it dependent on other repositories. Such repositories are\n"
+            "usually necessary for successful installation of RPMs from\n"
+            "the main Copr repository (runtime dependencies provided).\n\n"
+
+            "Be aware that the note about quality and bug-reporting\n"
+            "above applies here too, Fedora Project doesn't control the\n"
+            "content. Please review the list:\n\n"
+            "{0}\n\n"
+            "These repositories have been enabled automatically.\n\n"
+        )
+
+        counter = itertools.count(1)
+        sys.stderr.write(msg.format("\n\n".join([
+            "{num:2}. [{repoid}]\n    baseurl={baseurl}".format(
+                num=next(counter),
+                repoid=repoid,
+                baseurl=repo.cfg.getValue(repoid, "baseurl"))
+            for repoid in runtime_deps
+        ])))
```

Comment:
I'm not sure I find this more readable myself but I guess it's subjective :sunglasses:

Reply:
accepted